src/Link/__snapshots__/Link.spec.js.snap

```diff
@@ -0,0 +1,67 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Link default 1`] = `
+.c0 {
+  color: black;
+}
+
+.c0:visited {
+  color: black;
+}
+
+.c1 {
+  height: 1em;
+  margin: 0 0.5em;
+  vertical-align: top;
+}
+
+<span>
+  <a
+    className="c0"
+    color="black"
+    href="/"
+    target="_blank"
+  >
+    test link
+  </a>
+  <img
+    alt="link icon"
+    className="c1"
+    height="1em"
+    src="/images/link.png"
```

Comment:
any chance of an svg?

Reply:
I'll swap, maybe will remove the need for `staticDirs`